From eebad41b5351c927fe0e010060a571f4ea0eae1b Mon Sep 17 00:00:00 2001
From: Pankaj Bharadiya <pankaj.laxminarayan.bharadiya@intel.com>
Date: Fri, 7 Jul 2017 18:20:26 +0530
Subject: [PATCH 4174/4301] Fix compilation error

commit 6ad6ace617af2d391f9e5828b2a96fa746f63d1b from
git://git@github.com:01org/linux-intel-4.9.git

sound/soc/intel/skylake/skl-messages.c:36:28: fatal error: asm/set_memory.h: No such file or directory
compilation terminated.
scripts/Makefile.build:293: recipe for target 'sound/soc/intel/skylake/skl-messages.o' failed
make[4]: *** [sound/soc/intel/skylake/skl-messages.o] Error 1
scripts/Makefile.build:544: recipe for target 'sound/soc/intel/skylake' failed

Change-Id: Ie48f4d47d95ced73dff23028e377ee36657add50
Tracked-On:
Signed-off-by: Pankaj Bharadiya <pankaj.laxminarayan.bharadiya@intel.com>
---
 sound/soc/intel/skylake/cnl-sst.c      |    2 +-
 sound/soc/intel/skylake/skl-messages.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/intel/skylake/cnl-sst.c b/sound/soc/intel/skylake/cnl-sst.c
index e163907..1c29c8f 100644
--- a/sound/soc/intel/skylake/cnl-sst.c
+++ b/sound/soc/intel/skylake/cnl-sst.c
@@ -31,7 +31,7 @@
 #include <linux/device.h>
 #include <linux/sdw_bus.h>
 #include <linux/sdw/sdw_cnl.h>
-#include <asm/set_memory.h>
+#include <asm/cacheflush.h>
 
 #include "../common/sst-dsp.h"
 #include "../common/sst-dsp-priv.h"
diff --git a/sound/soc/intel/skylake/skl-messages.c b/sound/soc/intel/skylake/skl-messages.c
index 536b80e..79cb617 100644
--- a/sound/soc/intel/skylake/skl-messages.c
+++ b/sound/soc/intel/skylake/skl-messages.c
@@ -33,7 +33,7 @@
 #include "skl-tplg-interface.h"
 #include <linux/sdw/sdw_cnl.h>
 #include <linux/sdw_bus.h>
-#include <asm/set_memory.h>
+#include <asm/cacheflush.h>
 
 static int skl_alloc_dma_buf(struct device *dev,
 		struct snd_dma_buffer *dmab, size_t size)
-- 
1.7.5.4

